frontend/javascripts/oxalis/view/right-border-tabs/segments_tab/segment_statistics_modal.tsx

```diff
@@ -35,21 +50,25 @@ type SegmentInfo = {
   formattedSize: string;
   volumeInVoxel: number;
   boundingBoxTopLeft: Vector3;
+  boundingBoxTopLeftAsString: string;
   boundingBoxPosition: Vector3;
+  boundingBoxPositionAsString: string;
 };
 
 const exportStatisticsToCSV = (
-  segmentInformation: Array<SegmentInfo>,
+  segmentInformation: Array<SegmentInfo> | string,
   tracingId: string,
   groupIdToExport: number,
+  hasAdditionalCoords: boolean,
 ) => {
-  if (segmentInformation.length === 0) {
+  if (segmentInformation.length === 0 || segmentInformation instanceof String) {
```

Comment:
this if-statement could disappear then completely. I think, it's alright to emit an empty CSV if the table is empty. feels a bit more consistent than doing nothing at all.